packages/api/src/routes/helpers/request-logger.ts

```diff
@@ -1,9 +1,10 @@
 import { NextFunction, Request, Response } from "express";
-import { nanoid } from "nanoid";
+import { customAlphabet } from "nanoid";
 import { getLocalStorage } from "@metriport/core/util/local-storage";
 import { analyzeRoute } from "./request-analytics";
 
 const asyncLocalStorage = getLocalStorage("reqId");
+const nanoid = customAlphabet("0123456789abcdefghijklmnopqrstuvwxyz");
```

Comment:
nit: simplifies copy/past of req id on CloudWatch logs w/ double click (no special chars)